core/contracts/oracle/implementation/Store.sol

```diff
@@ -120,25 +121,28 @@ contract Store is StoreInterface, Withdrawable {
 
     /**
      * @notice Sets a new oracle fee per second.
-     * @param newOracleFee new fee per second charged to use the oracle.
+     * @param newOracleFeePerSecondPerPFC new fee per second charged to use the oracle.
      */
-    function setFixedOracleFeePerSecond(FixedPoint.Unsigned memory newOracleFee)
```

Comment:
In `scripts/PrecisionErrors.js` and `scripts/IntegrationTests.js`

Reply:
these changes were made in these scripts already :)